adaptors/library/jobs/satusehat-create-condition.js

```diff
@@ -0,0 +1,97 @@
+// Create a diagnosis in satusehat
+
+fn(state => {
```

Comment:
Maybe a comment like "Map diagnosis objects [from commcare?] into Condition resources", just to set some context